mailer.js

```diff
@@ -179,6 +181,45 @@ module.exports = function (log) {
     })
   }
 
+  Mailer.prototype.reVerifyEmail = function (message) {
```

Comment:
Please call this something like "confirmLoginEmail" rather than using the term "reverify", which we found to cause a fair bit of confusion around this feature last week.